spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1856,13 +1856,16 @@ use this in a webapp is to inject it into a void method in a
 
 [source,java,indent=0,subs="verbatim,quotes,attributes"]
 ----
-	@Configuration
+	@EnableWebSecurity
```

Comment:
This will switch off all of Boot's web security configuration, rather than just customising the authentication manager.